src/pilot/templates/first-boot.yaml

```diff
@@ -178,7 +178,7 @@ resources:
             echo $cmd
             $cmd
             echo
-
+            
```

Comment:
The extra spaces may be causing an issue...

Reply:
I checked with the previous files and even in this file that we are using spaces to make the code more readable but yes I will look into this.

Reply:
I think having extra blank lines is ok, but it may be that having any whitespace in a blank line is not.

Reply:
The ansible playbook issue was with the original first-boot.yaml file in which I added the kernel parameters. I checked and there doesnt seem to any extra whitespaces. I am gonna keep looking into it. Have restarted the deployment on R8. Lets see how that goes.